certified-connectors/Haiilo/apiDefinition.swagger.json

```diff
@@ -16,7 +21,7 @@
     "/publicapi/channel": {
       "get": {
         "description": "Retrieve channel list for creating a new post",
-        "operationId": "SmarpRetrieveChannelList",
+        "operationId": "AdvocacyRetrieveChannelList",
```

Comment:
Given that this is a significant alteration, could you kindly verify that this change will not impact any current flows or users? Additionally, please reach out to existing customers who have already established flows with this connector to notify them of the upcoming rebranding, as it represents a substantial modification.

Reply:
Hi, we've sent out communication-related to the branding update. Our customers are aware of the change and expecting it to be updated at anytime.